rotkehlchen/tests/api/test_makerdao_vaults.py

```diff
@@ -891,3 +894,48 @@ def test_query_vaults_usdc_strange(rotkehlchen_api_server, ethereum_accounts):
     details = assert_proper_response_with_result(response)
     expected_details = [vault_7538_details]
     assert_serialized_lists_equal(expected_details, details)
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_modules', [['makerdao_vaults']])
+@pytest.mark.parametrize('mocked_proxies', [{
+    TEST_ADDRESS_1: '0x689D4C2229717f877A644A0aAd742D67E5D0a2FB',
+}])
+@pytest.mark.parametrize('have_decoders', [True])
+def test_delete_vault_owner(rotkehlchen_api_server: 'APIServer'):
```

Comment:
for what do we need the decoders?

Reply:
Decoders are needed due to this https://github.com/rotki/rotki/blob/cc72e3aa01b6c81c0da71a5369d2db4934337e7c/rotkehlchen/rotkehlchen.py#L773.

The lock is needed